docs/upgrade/services_removal.md

```diff
@@ -0,0 +1,92 @@
+  ⮕ `ProductRepository::getSortedProductsByIds($domainId, PricingGroup $pricingGroup, array $sortedProductIds)`
+- `ProductService::markProductForVisibilityRecalculation(Product $product)`  
+  ⮕ `Product::markForVisibilityRecalculation()`
+- `ProductService::edit(Product $product, ProductData $productData)`  
+  ⮕ `Product::edit(ProductCategoryDomainFactoryInterface $productCategoryDomainFactory, ProductData $productData, ProductPriceRecalculationScheduler $productPriceRecalculationScheduler)`
+- `ProductService::delete(Product $product)`  
+  ⮕ `Product::getProductDeleteResult()`
+- `ProductService::recalculateInputPriceForNewVatPercent(Product $product, $productManualInputPrices, $newVatPercent)`  
+  ⮕ `ProductManualInputPrice::recalculateInputPriceForNewVatPercent($inputPriceType, $newVatPercent, BasePriceCalculation $basePriceCalculation, InputPriceCalculation $inputPriceCalculation)`
+- `OrderService::getOrderDetailUrl(Order $order)`  
+  ⮕ `OrderUrlGenerator::getOrderDetailUrl(Order $order)`
+- `OrderService:: createOrderProductInOrder(Order $order, Product $product, Price $productPrice)`  
+  ⮕ `Order::addProduct(Product $product, Price $productPrice, OrderProductFactoryInterface $orderProductFactory, Domain $domain, OrderPriceCalculation $orderPriceCalculation)`
+- `OrderService::editOrder(Order $order, OrderData $orderData)`  
+  ⮕ `Order::edit(OrderData $orderData, OrderItemPriceCalculation $orderItemPriceCalculation, OrderProductFactoryInterface $orderProductFactory, OrderPriceCalculation $orderPriceCalculation)`
+- `OrderStatusService::checkForDelete`  
+  ⮕ `OrderStatus::checkForDelete`
+- `OrderStatusService::createOrderProductInOrder`  
+  ⮕ `OrderStatus::addProductToOrder`
+
+Following classes have been removed:
+- `AdministratorService`
+- `AdministratorGridService`
+- `CustomerService`
+- `ProductService`
+- `OrderService`
+- `OrderStatusService`
+- `FlagService`
+
+Following methods have been removed:
+- `User::setDeliveryAddress()`, use `User::editDeliveryAddress()` instead
+- `Administrator::addGridLimit()`, use `Administrator::rememberGridLimit()` instead
+- `Administrator::removeGridLimit()` as it was not used anywhere
+- `Administrator::getLimitByGridId()` as it was not used anywhere
+- `FlagService::create()`, use `FlagFactory::create()` instead
+- `FlagService::edit()`, use `Flag::edit()` instead
+
+Following classes changed constructors:
+- `AdministratorFacade`
+- `AdministratorGridFacade`
+- `CachedBestsellingProductFacade`
+- `CustomerFacade`
+- `FlagFacade`
+- `OrderItemFacade`
+- `OrderProductFacade`
+- `OrderMailService`
+- `OrderEditResult`
+- `OrderFacade`
+- `ProductInputPriceFacade`
+- `ProductPriceRecalculator`
+- `ProductFacade`
+- `UserFactory`
+
+Following functions visibility was changed to `protected` as there is no need to use them from outside of objects:
+- `Administrator::getGridLimit()`
+- `Order::setTotalPrice()`
+
+Follow also unclassified upgrade instructions:
```

Comment:
There are some classified upgrade instructions? In meaning classified=secret :-) If you are describing that this instruction did not fit any category above, I would recommend term "additional" or something like this.